test/test.js

```diff
@@ -0,0 +1,49 @@
+var createS3UploadParams = require('../index').createS3UploadParams
```

Comment:
Codacy has a fix for the issue: [Replace `'../index').createS3UploadParams` with `"../index").createS3UploadParams;`](https://app.codacy.com/gh/dmgress/aws-ip-ranges/pullRequest?prid=9261990)
```suggestion
var createS3UploadParams = require("../index").createS3UploadParams;
```